### ISSUE TITLE
The optparse library is deprecated since Python v3.2

### ISSUE THREAD

**Issue by TobiasPleyer** [eliben/pss#55]
At some point _pss_ should use the _argparse_ library instead.

**Comment by TobiasPleyer** [comment]
I would solve this issue in one go with #54

**Comment by eliben** [comment]
Indeed, switching to argparse would be nice. Patches welcome, since I don't have time for it right now. It shouldn't be too hard but care advised - pss has so many command line options that the logic to reconcile them is a bit subtle
